soc/arm/atmel_sam/sam3x/dts_fixup.h

```diff
@@ -39,5 +39,6 @@
 #define DT_WDT_SAM_IRQ_PRIORITY DT_ATMEL_SAM_WATCHDOG_400E1A50_IRQ_0_PRIORITY
 #define DT_WDT_SAM_LABEL        DT_ATMEL_SAM_WATCHDOG_400E1A50_LABEL
 #define DT_WDT_SAM_BASE_ADDRESS DT_ATMEL_SAM_WATCHDOG_400E1A50_BASE_ADDRESS
+#define CONFIG_WDT_0_NAME       DT_WDT_SAM_LABEL
```

Comment:
Need to update prefix from `CONFIG_` to `DT_`

Reply:
Given the driver declares `HAS_DTS_WDT`, the `CONFIG_WDT_0_NAME` value is not filled by Kconfig and this has to be done in the fixup file. The same strategy is applied for the nRF, QMSI or MCUX drivers.